src/python/ddapp/teleoppanel.py

```diff
@@ -765,6 +768,15 @@ def searchFinalPoseClicked(self):
         if info == 1:
             self.panel.showPose(self.constraintSet.endPose)
         app.displaySnoptInfo(info)
+    
+    def onCandidateEndPose(self, msg):
+        if not self.getCheckboxState(self.ui.finalPosePlanningOptions) and not self.ui.eeTeleopButton.checked:
+            translation = (msg.pose.translation.x, msg.pose.translation.y, msg.pose.translation.z)
+            rotation = transformUtils.quaternionToRollPitchYaw((msg.pose.rotation.w, msg.pose.rotation.x, msg.pose.rotation.y, msg.pose.rotation.z))
+            pose = list(translation)
+            pose.extend(rotation)
+            pose.extend(list(msg.joint_position))
```

Comment:
this is not the correct way to construct a pose.  You should use:


pose = robotstate.convertStateMessageToDrakePose(msg)

That function will do the correct mapping from joint names to positions using the msg.joint_name array.